libraries/AdaptiveExpressions/BuiltinFunctions/ConvertFromUtc.cs

```diff
@@ -23,24 +25,30 @@ private static (object value, string error) Evaluator(Expression expression, IMe
             object value = null;
             string error = null;
             IReadOnlyList<object> args;
+            var locale = options.Locale != null ? new CultureInfo(options.Locale) : Thread.CurrentThread.CurrentCulture;
```

Comment:
Should we default to CurrentCulture or CurrentUICulture? See [this article](https://stackoverflow.com/questions/329033/what-is-the-difference-between-currentculture-and-currentuiculture-properties-of ) for additional details:

![image](https://user-images.githubusercontent.com/12264946/87802518-5abb7f00-c81f-11ea-9398-88a65b8f5e59.png)

Reply:
One of my concern this code is running on the server side. No matter CurrentCulture or CurrentUICulture was obtained from the server rather than reflecting the user's locale. 
If we have the user's culture info, it can be passed by the locale in incoming activities. The CurrentCulture used here is just a default value only if we could not determine user's locale.